src/lib.rs

```diff
@@ -383,11 +377,6 @@ impl Check {
                             })?;
                             Ok(true)
                         } else {
-                            config.shell_status(
-                                "Parsing",
-                                format_args!("{crate_name} v{version} (current)"),
-                            )?;
```

Comment:
This is duplicated here, since the subsequent `generate_versioned_crates()` call will log this information too.